README

```diff
@@ -61,3 +61,26 @@ devices:
     fastboot: 91671140
     fastboot_set_active: true
     fastboot_key_timeout: 2
+
+  - board: testboard
+    console: /dev/serial/by-id/usb-1234-if00-port0
+    name: GPIO controller board
+    local_gpio:
+      - type: power
+        chip: gpiochip0
+        line: 7
+        active_low: true
+      - type: fastboot_key
+        chip: gpiochip0
+        line: 8
+        active_low: true
+      - type: power_key
+        chip: gpiochip0
+        line: 14
+        active_low: true
+      - type: usb_disconnect
+        chip: gpiochip1
+        line: 4
```

Comment:
What would be your opinion about the following approach:

```yaml
local_gpio:
  power:
    chip: gpiochip0
    line: 7
    active_low: y
  fastboot_key:
    chip: gpiochip0
    line: 9    
```

Reply:
Indeed, it looks much better, I'll switch to that